stats/parsing-stats/lang/typescript/projects.txt

```diff
@@ -9,7 +9,7 @@
 #
 
 https://github.com/ant-design/ant-design
-https://github.com/angular/angular
+# https://github.com/angular/angular
```

Comment:
I agree with Nat. At least you need to explain why this was commented.
Like for skipping the OCaml repo in ../ocaml/projects.txt I've added a comment explaining why it was skipped.